spec.bs

```diff
@@ -171,10 +174,12 @@ are:
       1. [=list/Append=] |igAd| to |interestGroup|'s |interestGroupField|.
 1. If |interestGroup|'s [=interest group/estimated size=] is greater than 50 KB, then
   [=exception/throw=] a {{TypeError}}.
+1. Let |permission| be the result of [=checking interest group permissions=] with 

```

Comment:
maybe we can move this to where the .well-known todo was, i.e., inside the parallel queue?

Reply:
At least two of those parallel steps should not be performed if this check fails, so I think they need to be done sequentially.

Reply:
oh I think spec's "in parallel" term is misleading. Steps in "in parallel" means they will not block the main process and will run in parallel to steps outside of "in parallel" block, but these steps inside "in parallel" block will run sequentially. See https://github.com/WICG/turtledove/pull/488#discussion_r1149555225

Reply:
OK, sounds good.  I think it will be a little more complicated since most of those steps don't run if the check fails, but I think there are examples of how to handle that.

Reply:
Done.